.gitignore

```diff
@@ -59,4 +59,4 @@ build/
 .vscode/
 
 ### env ###
```

Comment:
Marco, can you try to cd to the repo directory and set `git config core.autocrlf true`

And then commit again. Hopefully, the pull request won't show this difference.